README.md

```diff
@@ -101,3 +124,22 @@ Server logs will be as below:
 [Plugin [element-wait (sessionless)]] Checking if login element is displayed
 [Plugin [element-wait (sessionless)]] login element is displayed.
 ```
+
+## Skip element Enabled Check for commands
+Before performing actions such as `click`, `setValue`, `clear` etc, plugin waits for element to be enabled.
+As mentioned in [#78](https://github.com/AppiumTestDistribution/appium-wait-plugin/issues/78) if there is a need to skip the `elementEnabled` check for a set of commands, `excludeEnabledCheck` can be sent along with timeout values. 
+##### Usage
+wdio
+```
+await driver.executeScript('plugin: setWaitPluginProperties', [
+        {
+          timeout: 1111,
+          intervalBetweenAttempts: 11,
+		  excludeEnabledCheck: ['click','setValue']
+        },
+]);
+
+await driver.executeScript('plugin: getWaitPluginProperties', [])
```

Comment:
Is empty array mandatory here?

Reply:
@SrinivasanTarget Yes, thats executeScript contract.